distributed-job-manager/src/main/java/ru/fix/distributed/job/manager/Manager.java

```diff
@@ -86,6 +99,49 @@ public void start() throws Exception {
 
         leaderLatch.start();
         workersAliveChildrenCache.start();
+        startWorkPoolCleaningTask();
+    }
+
+    private void startWorkPoolCleaningTask() {
+            workPoolCleaningReschedulableScheduler.schedule(
+                    DynamicProperty.delegated(() -> Schedule.withDelay(workPoolCleanPeriodMs.get())),
```

Comment:
Could you please check javadoc on delegated method. You did exactly what you must not do.  Listener will not be invoked on property change.

Reply:
fixed